test/e2e/provisioning/testdata/machinedeployment-openstack.yaml

```diff
@@ -34,7 +34,6 @@ spec:
             domainName: "<< DOMAIN_NAME >>"
             region: "<< REGION >>"
             network: "<< NETWORK_NAME >>"
-            rootDiskSizeGB: 20
```

Comment:
isn't this supposed to be changed to 10 instead of being completely removed?

Reply:
We have two manifests we use for openstack. I wanted to have 1 in which we specify `rootDiskSizeGB` and one where we do not in order to cover both options. I moved it to the other one because we run it just with ubuntu, while this one is used many times. Not sure actually if without specifying anything we consume more or less disk, I need to double check this.